app/scripts/main.js

```diff
@@ -74,15 +74,26 @@ $(document).ready(function () {
         };
     };
 
-    var openManPage = function () {
+    var openManPage = function (selectedManPage) {
         var openManPageTabs;
-        if (lastSelectedManPage) {
+        if (selectedManPage) {
             openManPageTabs = viewModel.openManPageTabs;
-            openManPageTabs.push(getManPageTabData(lastSelectedManPage));
+            openManPageTabs.push(getManPageTabData(selectedManPage));
             viewModel.currentActiveTabIndex(openManPageTabs().length - 1);
         }
     };
 
+    var manPageTokens = new Bloodhound({
+        datumTokenizer: Bloodhound.tokenizers.obj.whitespace('name', 'summary'),
+        queryTokenizer: Bloodhound.tokenizers.whitespace,
+        limit: 10,
+        prefetch: {
+            url: 'sysassets/man_pages/sys_man_page_index.min.json'
```

Comment:
I tried it locally and as much as I could observe it doesn't download it twice but uses it from the browser cache probably because the URL is the same. Also, as far as I could see I couldn't even see any network request in the console to this URL but it does create a key in local storage which I guess is used for any further data request.